src/app/clusters/ota-requestor/BDXDownloader.cpp

```diff
@@ -131,7 +134,7 @@ void BDXDownloader::EndDownload(CHIP_ERROR reason)
         {
             mImageProcessor->Abort();
         }
-        SetState(State::kIdle);
+        SetState(State::kIdle, OTAChangeReasonEnum::kSuccess);
```

Comment:
`SetState` eventually calls into OTA Requestor core to let it know the download state. When the download is ending (but not complete yet), it uses the same `kIdle` state but the core logic needs a way to differentiate when this is a successful end vs one of the above failure cases so I added in an extra param to help with that.